x-pack/plugins/security_solution/server/endpoint/lib/artifacts/common.ts

```diff
@@ -7,13 +7,13 @@ import { Logger } from 'src/core/server';
 
 export const ArtifactConstants = {
   GLOBAL_ALLOWLIST_NAME: 'endpoint-exceptionlist',
-  SAVED_OBJECT_TYPE: 'endpoint:user-artifact:v2',
+  SAVED_OBJECT_TYPE: 'endpoint:user-artifact',
```

Comment:
just curious, why is the `v2` being removed?

Reply:
@marshallmain It was just there to help with testing during dev... things were changing quickly and people were not always starting with a fresh install. I had gotten some feedback on it, and I think this is a good time to remove it before we go down the path toward GA.